### DISCUSSION TITLE
Add a "Temporarily Highlighted Items" List in the Ground Items plugin

### DISCUSSION THREAD

**Discussion by vanoreo** [runelite/runelite#15513] [Ideas]
**Is your feature request related to a problem? Please describe.**
Occasionally it can be useful to highlight specific items situationally, whether you be juggling or doing some task. It would be nice to have a separate area to set highlighted items that can easily be toggled as a group depending on the situation.

**Describe the solution you'd like**
Another group with an associated checkbox, that when checked, gets added to the standard "Highlighted Items" list, and when unchecked, does not.

**Describe alternatives you've considered**
The current solution is to manually add/remove each item as is necessary, which is somewhat annoying

**Comment by Azeirah** [comment]
An alternative solution to this would be to allow you to save settings set-ups.

If you can 

- create and name
- duplicate
- name
- remove
- reorder

A configuration, then you can do this and much more.

For example, I wouldn't mind having a set-up ignoring "Raw beef" and highlighting "cow hide" when farming cows. I'd call this setting "Farming cowhide"
